extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1358,7 +1372,8 @@ export class AmpA4A extends AMP.BaseElement {
    * @private
    */
   renderViaNameAttrOfXOriginIframe_(creativeBody) {
-    const method = this.experimentalNonAmpCreativeRenderMethod_;
+    const method = (this.isFluid && XORIGIN_MODE.SAFEFRAME) ||
```

Comment:
Nit: comment and closure type would be useful here

Reply:
Added.